### PR TITLE
task-wLxLFVi0 

### PR DESCRIPTION
**Pull request by AntonShalenui**
https://trello.com/c/wLxLFVi0/33-info-add-form
add form in Info component,Add input, textarea and button.

### CODE REVIEW

File: src/client/components/Info.js [hunk 1]
```diff
@@ -3,9 +3,20 @@ import './info.css';
 
 export default class Info extends Component {
   render() {
+    function View() {
+    }
```
Comment (vkryviak-ua) [inline src/client/components/Info.js]: Please remove this function. You have no requiremets for this

File: src/client/components/Info.js [hunk 2]
```diff
@@ -3,9 +3,20 @@ import './info.css';
...
     return (
       <aside className="global-info">
         New component Info.
+      <div>
+        <h4>Name</h4>
```
Comment (vkryviak-ua) [inline src/client/components/Info.js]: Probably will be good to have each element in div and no h4
<div>Name</div>
<div><input ....

Comment (AntonShalenui) [reply src/client/components/Info.js]: ok

File: src/client/components/Info.js [hunk 3]
```diff
@@ -6,6 +6,11 @@ export default class Info extends Component {
     return (
       <aside className="global-info">
         New component Info.
+      <div>Name</div>
+      <div><input></input></div>
+      <div>Description</div>
+      <div><textarea></textarea></div>
+      <div><button onClick={this.View}> View </button></div>
```
Comment (vkryviak-ua) [inline src/client/components/Info.js]: please remove onclick

File: src/client/components/Info.js [hunk 4]
```diff
@@ -6,6 +6,11 @@ export default class Info extends Component {
     return (
       <aside className="global-info">
         New component Info.
+      <div>Name</div>
+      <div><input></input></div>
```
Comment (vkryviak-ua) [inline src/client/components/Info.js]: Input has no close tag

**Pull request merged**